tests/cli-tests/migrations.test.ts

```diff
@@ -306,7 +306,6 @@ CREATE TABLE "Todo" (
 Prompt: Name of migration init
 ? A migration has been created at migrations/migration_name
 Prompt: Would you like to apply this migration? true
-Applying migration \`migration_name\`
```

Comment:
This isn't being captured by `jest` at this time, but it is still printing, I'll follow that up in a different way - it doesn't need to block